frontend/src/components/pages/reassign-partitions/components/ActiveReassignments.tsx

```diff
@@ -10,22 +10,30 @@
  */
 
 import React, { Component, FC, useRef } from 'react';
-import { Modal, Popconfirm, Skeleton } from 'antd';
+import { Popconfirm, Skeleton } from 'antd';
 import { ConfigEntry } from '../../../../state/restInterfaces';
 import { api } from '../../../../state/backendApi';
 import { computed, makeObservable, observable } from 'mobx';
 import { QuickTable } from '../../../../utils/tsxUtils';
 import { sortField } from '../../../misc/common';
 import { uiSettings } from '../../../../state/ui';
-import { Message, prettyBytesOrNA, prettyMilliseconds } from '../../../../utils/utils';
+import { prettyBytesOrNA, prettyMilliseconds } from '../../../../utils/utils';
 import { ReassignmentState } from '../logic/reassignmentTracker';
 import { observer, useLocalObservable } from 'mobx-react';
 import { reassignmentTracker } from '../ReassignPartitions';
 import { BandwidthSlider } from './BandwidthSlider';
 import { KowlColumnType, KowlTable } from '../../../misc/KowlTable';
 import { BrokerList } from '../../../misc/BrokerList';
-import { Button, Checkbox, Progress, ToastId, useToast } from '@redpanda-data/ui';
-
+import { Box, Button, Checkbox, createStandaloneToast, Flex, ListItem, Modal, ModalBody, ModalContent, ModalFooter, ModalHeader, ModalOverlay, Progress, redpandaToastOptions, Text, ToastId, UnorderedList, useToast } from '@redpanda-data/ui';
+
+// TODO - once ActiveReassignments is migrated to FC, we could should move this code to use useToast()
+const { ToastContainer, toast } = createStandaloneToast({
+    defaultOptions: {
+        ...redpandaToastOptions.defaultOptions,
+        isClosable: false,
```

Comment:
[question] Is it expected that the `isClosable` is set to false? Shouldn't we let the user dismiss the toast. Not sure how Console usually handles it.

Reply:
I tried to make "messages" non closable and toast closable.